translations/messages.fr.xlf

```diff
@@ -0,0 +1,3061 @@
+        The light group here only has one freedom left, capture it.</source>
+        <target>Des pierres connectées horizontalement ou verticalement doivent être capturées ensemble, et ne sont pas capturables séparement.&lt;br/&gt;&lt;br/&gt;
+        Ici, le groupe clair n&apos;a plus qu&apos;une liberté, capturez ce groupe.</target>
+      </trans-unit>
+      <trans-unit id="2022880801532921915" datatype="html">
+        <source>Congratulations, you have earned three points and formed a territory.</source>
+        <target>Bravo, vous avez gagné trois points, et formé un territoire.</target>
+      </trans-unit>
+      <trans-unit id="682384900189815989" datatype="html">
+        <source>Failed, you have not captured the group. Play on the last freedom of that group.</source>
+        <target>Raté, vous n&apos;avez pas capturé le groupe, jouez sur la dernière liberté de ce groupe.</target>
+      </trans-unit>
+      <trans-unit id="6220902431017372113" datatype="html">
+        <source>Suicide</source>
+        <target>Suicide</target>
+      </trans-unit>
+      <trans-unit id="1598565563312868670" datatype="html">
+        <source>In Go, suicide is forbidden.
+        If putting a piece on an intersection removes the last freedom of your group and does not capture any stone, playing on that intersection would be a suicide and is therefore forbidden.
+        Here, the top left intersection is a suicide for Light.
+        On the bottom right, it would be a suicide for Dark, and on the bottom left it is not a suicide for any player.</source>
+        <target>Au Go le suicide est interdit.
+        Quand mettre une pierre sur une intersection ferait que le groupe de votre dernière pierre n&apos;a aucune liberté et ne capture aucunes pierres, jouer cette intersection serait un suicide, et est donc interdit.
+        Ici, l&apos;intersection en haut à gauche est un suicide pour Clair.
+        En bas à droite, un suicide pour Foncé, et en bas à gauche n&apos;est un suicide pour aucun joueur.</target>
+      </trans-unit>
+      <trans-unit id="2066383177849177665" datatype="html">
+        <source>Life and death (death)</source>
+        <target>Vie et mort (mort)</target>
+      </trans-unit>
+      <trans-unit id="5995146621586456333" datatype="html">
+        <source>From the capture rule follows the life and death notion:
+        dead stones are stones that are definitely capturable (without losing anything else).
+        Alive stones are stones that can never be captured.
+        From the capture rule, Dark can play inside Light&apos;s territory and make a capture.
+        In this case, we say that Light has only one eye (its last freedom) and that Light is dead (even if not yet captured).
+        At the end of the game, the dead stone will count as a capture, and the intersection it occupies as a territory.</source>
```

Comment:
the dead**s** stone**s** will count as capture**s**, and the intersection they occupy(?) as territories. 

?

Reply:
oui, sauf pour le s à dead